projects/angular/src/popover/signpost/_signposts.clarity.scss

```diff
@@ -366,11 +366,15 @@
 
   .signpost-content-header {
     display: flex;
-    justify-content: flex-end;
+    justify-content: space-between;
     width: 100%;
     background-color: inherit;
     top: 0;
     border-radius: signpost-variables.$clr-signpost-border-radius;
+
+    .clr-signpost-title {
+      @include mixins.generate-typography-token('SECTION-12-SB-CPT');
```

Comment:
Not consistent with Figma spec. I see '/* *ColumnHeader-11-SB-CPT */' there. Check with design.

Reply:
I don't see ColumnHeader-11-SB-CPT token in code, so i updated with the TableHeaders-11-SB-CPT